apps/video/test/marionette/play_multiple_types_test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var assert = require('assert'),
+    Marionette = require('marionette-client'),
+    Video = require('./lib/video');
+
+marionette('play various types of videos', function() {
+
+  var app, client, actions, selectors;
+
+  client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    }
+  });
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    selectors = Video.Selector;
+    app = new Video(client);
+    actions = new Marionette.Actions(client);
+  });
+
+  test ('test various videos', function() {
+    var types = ['mp4', 'ogg', 'ogv', 'webm'];
+
+    types.forEach(function(type) {
+      var filename = 'test_' + type + '_video.' + type;
+
+      client.fileManager.add({
+        type: 'videos',
+        filePath: 'apps/video/test/videos/' + filename
+      });
+    });
+
+    app.launch();
+    app.waitForThumbnails(types.length);
+    var thumbnails = app.getThumbnails();
+
+    for (var i = 0; i < thumbnails.length; i++) {
+
+      app.tapThumbnail(i);
+      app.waitForControls();
```

Comment:
I like the idea to leave the function as it is and to rename it as you suggest. Thanks for that suggestion.